platform/linux-generic/include/odp_queue_basic_internal.h

```diff
@@ -54,6 +54,7 @@ struct queue_entry_s {
 		ring_spsc_t  ring_spsc;
 	};
 
+	odp_atomic_u64_t     timers;
```

Comment:
I think it's better to name it timers_cnt or timers_num.

Reply:
or timers_ref_cnt

Reply:
OK, will use use a more descriptive name.

Reply:
Looks like this name change is still pending?